### DISCUSSION TITLE
Graph database type?

### DISCUSSION THREAD

**Discussion by Jon-Biz** [orbitdb/orbit-db#1205] [Ideas]
I have a graph database implementation that I've been working with. Would it make sense to submit a PR for it? I'm thinking it would save others from needing to implement their own custom solutions.

**Comment by haydenyoung** [comment]
Thanks for your offer @Jon-Biz and we would definitely like to include it as part of OrbitDB. Instead of submitting it as a PR against the core OrbitDB repo, would you be able to make it a separate module? See [here](https://github.com/orbitdb/set-db) for an example.

I have also reached out to you on Matrix to discuss other particulars.

**Comment by simonfelding** [comment]
@Jon-Biz that would be really cool! Please do.
